connector/src/thegraph/queries/index.ts

```diff
@@ -1,8 +1,10 @@
 import gql from 'graphql-tag'
 
 export const CONFIG = (type: string) => gql`
-  ${type} Config($id: String!) {
-    config(id: $id) {
+  ${type} Configs($id: String!) {
+    configs(where: {
+      id: $id
+    }) {
```

Comment:
Did it not work with the previous query?

Reply:
nop!